packages/components/src/ComboBox.tsx

```diff
@@ -187,7 +194,9 @@ class ComboBox extends Component<ComboBoxProps, ComboBoxState> {
 
     switch (event.key) {
       case 'Enter':
-        this.updateInputValue(menuOptions[keyboardOptionIndex].value);
+        if (menuOptions[keyboardOptionIndex]?.value) {
```

Comment:
Should check for != null here, the value could be '' (empty string, I think?)

Reply:
Yep. Good catch